vm/src/vm/vm_core.rs

```diff
@@ -942,6 +942,17 @@ impl VirtualMachine {
 
         Err(VirtualMachineError::NoSignatureBuiltin)
     }
+
+    pub fn get_output_builtin(&mut self) -> Result<&mut OutputBuiltinRunner, VirtualMachineError> {
```

Comment:
Can we name it `get_output_builtin_mut` instead? As we have similar named methods that return immutable references to other builtins and it may be confusing

Reply:
Fixed.